boards/intel/btl/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
soc commit before board commit

Reply:
Board Config used in soc commit is from board definition, so I added board first and then added soc changes for the board.